generated/python/gapic-google-cloud-functions-v1beta2/README.rst

```diff
@@ -39,40 +72,7 @@ Windows
     pip install virtualenv
     virtualenv <your-env>
     <your-env>\Scripts\activate
-    <your-env>\Scripts\pip.exe install gapic-google-cloud-speech-v1beta1
+    <your-env>\Scripts\pip.exe install google-cloud-functions-v1beta2
```

Comment:
This should be `gapic-google-cloud-functions-v1beta2`, which should have been fixed by googleapis/toolkit#932. Could you regenerate with the latest version of `toolkit` and update the PR?